code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -24,7 +24,7 @@
 	can_suppress = 0
 
 /obj/item/weapon/gun/projectile/automatic/pistol/deagle
-	name = "desert eagle"
+	name = "\improper Desert Eagle"

```

Comment:
Well, considering I am doing it for the other guns, why not, it would be consistent.